src/components/organisms/UiMultipleAnswer/_internal/UiMultipleAnswerItem.vue

```diff
@@ -215,10 +253,37 @@ if (choiceItem.value) {
   $this: &;
   $element: multiple-answer-item;
 
+  &--has-info {
+    #{$this}__label {
```

Comment:
this after element allow us to keep required spacings (for example margin) for suffix element